.cirrus.yml

```diff
@@ -577,6 +584,15 @@ task:
         - ulimit -S -n 2048 # https://github.com/flutter/flutter/issues/2976
         - ./dev/bots/deploy_gallery.sh
 
+    - name: verify_binaries_codesigned-macos # macos-only
+      # TODO(fujino): remove this `only_if` after https://github.com/flutter/flutter/issues/44372
+      only_if: "$CIRRUS_BRANCH == 'dev' || $CIRRUS_BRANCH == 'beta' || $CIRRUS_BRANCH == 'stable' || $CIRRUS_BRANCH =~ '.*hotfix.*'"
```

Comment:
Why didn't verify_binaries_codesigned-macos run on this PR?  We won't know if it passed until this is merged into stable, which is too late...

Reply:
I tested this by commenting out the conditional and running the test. The reason it didn't run as is is because $CIRRUS_BRANCH is something like `pull/50291` pre-submit. We'd catch a failure before before it landed in stable because hotfixes dont' get merged to stable until after CI is green on the hotfix branch.

Reply:
Oh, but even better, I can check `$CIRRUS_BASE_BRANCH != 'master'`...

Reply:
Will that catch random PRs off master that aren't codesigned too?

Reply:
Well, the githubbot should change those to be on master. Not sure if the bot changing the base will re-trigger the cirrus jobs.